build.gradle.kts

```diff
@@ -30,7 +42,16 @@ spotless {
     }
 }
 
-tasks.withType<JavaCompile>().configureEach { options.encoding = "utf-8" }
+tasks.withType<JavaCompile>().configureEach { 
+    options.encoding = "utf-8"
+    options.errorprone {
+            disableAllChecks.set(true)
+            error(
+                "MissingOverride",
+                "WildcardImport"
+            )
+        }
+    }
```

Comment:
nit: could be shifted left. (You could configure Spotless to check the build file too.)